assets/src/edit-story/components/rangeInput/index.js

```diff
@@ -58,4 +64,52 @@ export default styled.input.attrs(({ min, max, step, value, onChange }) => ({
   &::-ms-thumb {
     ${rangeThumb}
   }
+
+  &:focus {
+    &::-webkit-slider-thumb {
+      ${focusedRangeThumb}
+    }
+
+    &::-moz-range-thumb {
+      ${focusedRangeThumb}
+    }
+
+    &::-ms-thumb {
+      ${focusedRangeThumb}
+    }
+  }
 `;
+
+function RangeInput({ minorStep, majorStep, handleChange, value, ...rest }) {
```

Comment:
The `majorStep` is basically the default step, is that correct? The naming feels a bit odd as if there would be no option to do a "normal" step -- it's always either minor or major. WDYT if it would be just `step` and `minorStep`?
Or perhaps alternatively just leave a comment for clarity that `majorStep` is the default step?

Reply:
`majorStep` is the default when using keyboard, however when using the actual range, the "default" step (the smallest interval you can move with the mouse) would be the `minorStep`. I thought it was fairly self-explanatory but clearly not - will add some commentary.

Reply:
Oh, right, I was only focusing on the keyboard part and didn't even think about the mouse while reviewing.